source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0062
+
+<b>Functional Description</b>
+
+The rules R0506 and R0507 are applied on CC015C and CC013C to ensure that the declaration does not include unnecessary and repetitive information. They must be enforced by all NTA. Considering the possibility that one Goods Item is taken out from the declaration during the control, the message CC029C and CD001C may have different content from CC015C (or CC013C or CC170C). Consequently, those rules R0506 and R0507 shall not be strictly enforced on the Common Domain messages. Certainly not by the recipient of the CD message, likely not by the sender of the CD message.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0064
+
+<b>Functional Description</b>
+
+The message must be sent within thirteen (13) days from the day the “Destination Control Results” C_DES_CON (IE018) message is sent to the Office of Departure
+
+<b>Technical Description</b>
+
+N/A
+
+##G0068
+
+<b>Functional Description</b>
+
+The Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE&gt; will be also used to include the information of EMCS consignment exported from one EU member state into a Non-EU-Member state, in case of Export Followed by Transit (where in messages CC013C or CC015C the &lt;CONSIGNMENT- HOUSE CONSIGNMENT- PREVIOUS DOCUMENT.Type&gt; = ‘N830’ AND &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE. Type&gt; is in SET CL234 (DocumentTypeExcise)).<br/>In this case, the Data Group &lt;GOODS SHIPMENT- GOODS ITEM- PREVIOUS DOCUMENT&gt; of the Export declaration, will be mapped with the Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE&gt; of the Transit declaration.   
+
+<b>Technical Description</b>
+
+N/A
+
+##G0069
+
+<b>Functional Description</b>
+
+The Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- SUPPORTING DOCUMENT&gt;, can be also used to include the information related to EMCS consignment (where &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- SUPPORTING DOCUMENT.Type&gt; is in SET CL234 (DocumentTypeExcise)), transported from one EU member state into another EU member state via a CTC country.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0071
+
+<b>Functional Description</b>
+
+In case of Export Followed by Transit (i.e. &lt;CONSIGNMENT-HOUSE CONSIGNMENT- PREVIOUS DOCUMENT. Type&gt; = ‘N830’), all and only the goods items declared in &lt;GOODS SHIPMENT-GOODS ITEM&gt; as defined in the related Export declaration (identified by the MRN) must be included in &lt;CONSIGNMENT-HOUSE CONSIGNMENT- CONSIGNMENT ITEM&gt; Data Group.
+
```

Comment:
Two issues - 
- Unless `Export Followed by Transit` is phrasing that has to be preserved, suggest changing `In case of Export Followed` to `If Export is Followed`.
- Change `i.e.` to `that is,` (note comma).